src/k8s/cmd/k8s/k8s_join_cluster.go

```diff
@@ -76,5 +87,15 @@ func newJoinClusterCmd(env cmdutil.ExecutionEnvironment) *cobra.Command {
 	}
 	cmd.Flags().StringVar(&opts.name, "name", "", "node name, defaults to hostname")
 	cmd.Flags().StringVar(&opts.address, "address", "", "microcluster address, defaults to the node IP address")
+	cmd.PersistentFlags().StringVar(&opts.configFile, "file", "", "path to the YAML file containing your custom cluster join configuration")
```

Comment:
PersistentFlags are meant for flags that will be re-used by subcommands. Let's just do flags instead, to avoid possible future confusion.

```suggestion
	cmd.Flags().StringVar(&opts.configFile, "file", "", "path to the YAML file containing your custom cluster join configuration")
```